src/core/lib/iomgr/ev_epoll1_linux.c

```diff
@@ -154,9 +154,9 @@ struct grpc_pollset_worker {
   grpc_closure_list schedule_on_end_work;
 };
 
-#define SET_KICK_STATE(worker, state)        \
+#define SET_KICK_STATE(worker, kick_state)   \
```

Comment:
Why change the parameter name?

Reply:
Need to differentiate between the field name 'state' and macro parameter 'kick_state', otherwise it would try to find worker->KICKED instead of worker->state